pmm/pmm2-api-tests.groovy

```diff
@@ -0,0 +1,98 @@
+void runStaging(String DOCKER_VERSION, CLIENT_VERSION, CLIENTS) {
+    stagingJob = build job: 'aws-staging-start', parameters: [
+        string(name: 'DOCKER_VERSION', value: DOCKER_VERSION),
+        string(name: 'CLIENT_VERSION', value: CLIENT_VERSION),
+        string(name: 'CLIENTS', value: CLIENTS),
+        string(name: 'NOTIFY', value: 'false'),
+        string(name: 'DAYS', value: '1')
+    ]
+    env.VM_IP = stagingJob.buildVariables.IP
+    env.VM_NAME = stagingJob.buildVariables.VM_NAME
+    env.PMM_URL = "http://${VM_IP}"
+}
+
+void destroyStaging(IP) {
+    build job: 'aws-staging-stop', parameters: [
+        string(name: 'VM', value: IP),
+    ]
+}
+
+pipeline {
+    agent {
+        label 'micro-amazon'
+    }
+    parameters {
+        string(
+            defaultValue: 'master',
+            description: 'Tag/Branch for pmm-api-tests repository',
+            name: 'GIT_BRANCH')
+        string(
+            defaultValue: 'perconalab/pmm-server:dev-latest',
+            description: 'PMM Server docker container version (image-name:version-tag)',
+            name: 'DOCKER_VERSION')
+        string(
+            defaultValue: 'dev-latest',
+            description: 'PMM Client version',
+            name: 'CLIENT_VERSION')
+    }
+    options {
+        skipDefaultCheckout()
+        disableConcurrentBuilds()
+    }
+    triggers {
+        upstream upstreamProjects: '', threshold: hudson.model.Result.SUCCESS
+    }
+
+    stages {
+        stage('Prepare') {
+            steps {
+                // clean up workspace and fetch pmm-qa repository
```

Comment:
But that's not pmm-qa repo

Reply:
it will be in the future, once we merge changes to api-tests

Reply:
But then we can update comment too. I don't think that is a good idea to have a comment in sync with _future_ code, not with the current one. :)